polyphemus/lib/ipi/migrations/01_flow_model_migration.rb

```diff
@@ -174,6 +174,10 @@ def format_attribute_value(suffix, value)
     # So even though we don't know the original Metis
     #   path or file name, we can still create a copy of the
     #   Magma copy of the file.
+    return {
```

Comment:
This is not strictly necessary since we aren't copying old data into the new `flow` model.